### PR TITLE
build/config/rtl8721csm: add ipv6 config

### PR DESCRIPTION
**Pull request by pillip8282**
add ipv6 config

### CODE REVIEW

File: build/configs/rtl8721csm/ipv6/Make.defs [hunk 1]
```diff
@@ -0,0 +1,228 @@
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
```
Comment (jeongchanKim) [inline build/configs/rtl8721csm/ipv6/Make.defs:3]: Please change to 2021.

**Review by an4967 — DISMISSED** [review]
LGTM

File: build/configs/rtl8721csm/ipv6/defconfig [hunk 2]
```diff
@@ -0,0 +1,1348 @@
...
+#
+# Build Configuration
+#
+CONFIG_APPS_DIR="../apps"
+CONFIG_FRAMEWORK_DIR="../framework"
+CONFIG_TOOLS_DIR="../tools"
+CONFIG_BUILD_FLAT=y
+# CONFIG_APP_BINARY_SEPARATION is not set
+# CONFIG_BUILD_2PASS is not set
+CONFIG_BOARD_BUILD_DATE=200204
```
Comment (jeongchanKim) [inline build/configs/rtl8721csm/ipv6/defconfig:26]: For this PR(https://github.com/Samsung/TizenRT/pull/5014), 
Please change to `CONFIG_BOARD_BUILD_DATE="200204"`

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**